lib/Alchemy/Phrasea/Controller/Prod/PushController.php

```diff
@@ -597,7 +597,7 @@ public function editListAction(Request $request, $list_id)
 
     private function formatUser(User $user)
     {
-        $subtitle = array_filter([$user->getJob(), $user->getCompany()]);
+        $subtitle = array_filter([htmlspecialchars($user->getJob()), htmlspecialchars($user->getCompany())]);
```

Comment:
No. A method in the controller does not have to know what usage is made of the result (html), this is the job of the final consumer to format for his own usage.
Here "formatUser" is called by a few public methods that return json, json should not contain html.
Html escaping should be done at render-time, usually in twig.